examples/console/main.go

```diff
@@ -325,197 +323,238 @@ func run() error {
+				SourceAccount:        &account,
+				IncrementSequenceNum: true,
+				BaseFee:              txnbuild.MinBaseFee,
+				Timebounds:           txnbuild.NewTimeout(300),
+				Operations: []txnbuild.Operation{
+					&txnbuild.Payment{Destination: destination.Address(), Asset: asset.Asset(), Amount: depositAmountStr},
+				},
+			})
+			if err != nil {
+				c.Err(fmt.Errorf("building deposit payment tx: %w", err))
+				return
+			}
+			tx, err = tx.Sign(networkPassphrase, signer)
+			if err != nil {
+				c.Err(fmt.Errorf("signing deposit payment tx: %w", err))
+				return
+			}
+			_, err = horizonClient.SubmitTransaction(tx)
+			if err != nil {
+				c.Err(fmt.Errorf("submitting deposit payment tx: %w", err))
+				return
+			}
+		},
+	})
+	shell.AddCmd(&ishell.Cmd{
+		Name: "pay",
+		Help: "pay <amount> - pay amount of asset to peer",
+		Func: func(c *ishell.Context) {
+			amt, err := amount.ParseInt64(c.Args[0])
+			if err != nil {
+				c.Err(err)
+				return
+			}
+			stats.Reset()
+			_, err = agent.Payment(amt)
+			c.Err(err)
+			return
+		},
+	})
+	shell.AddCmd(&ishell.Cmd{
+		Name: "payx",
+		Help: "payx <max-amount> <number of times> <max buffer size> - pay an amount a certain number of times",
+		Func: func(c *ishell.Context) {
+			fmt.Fprintf(os.Stdout, "sending %s payment %s times\n", c.Args[0], c.Args[1])
+			amt, err := amount.ParseInt64(c.Args[0])
+			if err != nil {
+				c.Err(err)
+				return
+			}
+			x, err := strconv.Atoi(c.Args[1])
+			if err != nil {
+				c.Err(err)
+				return
+			}
+			bufferSize, err := strconv.Atoi(c.Args[2])
+			if err != nil {
+				c.Err(err)
+				return
+			}
+			stats.Reset()
+			agent.SetMaxBufferSize(bufferSize)
+			stats.MarkStart()
+			amtRange := int64(bufferSize)
+			if amtRange > amt {
+				amtRange = amt
+			}
+			for i := 0; i < x; i++ {
+				memo := "tx-" + strconv.Itoa(i)
+				for {
+					amt := amt - (int64(i) % amtRange)
+					_, err = agent.PaymentWithMemo(amt, memo)
+					if err != nil {
+						continue
+					}
+					break
 				}
-				break
 			}
-		}
-		agent.Wait()
-		for stats.bufferedPaymentsSent != int64(x) {
-			// Wait for all the buffered payments to have been sent.
-		}
-		stats.MarkFinish()
-		fmt.Println(stats.Summary())
-		agent.SetMaxBufferSize(1)
-		return err
-	case "declareclose":
-		return agent.DeclareClose()
-	case "close":
-		return agent.Close()
-	case "listagreements":
-		for i, a := range closeAgreements {
-			var sender string
-			if signer.FromAddress().Equal(a.Envelope.Details.ProposingSigner) {
-				sender = "me"
-			} else {
-				sender = "them"
+			agent.Wait()
+			for stats.bufferedPaymentsSent != int64(x) {
+				// Wait for all the buffered payments to have been sent.
 			}
-			var receiver string
-			if signer.FromAddress().Equal(a.Envelope.Details.ConfirmingSigner) {
-				receiver = "me"
-			} else {
-				receiver = "them"
+			stats.MarkFinish()
+			fmt.Println(stats.Summary())
+			agent.SetMaxBufferSize(1)
+			c.Err(err)
+			return
```

Comment:
This return is redundant?

Reply:
yup lint caught it. Was trying to make it consistent, but removed
https://github.com/stellar/starlight/pull/403/commits/5b8e9b1c42912817a5e6a2f91d1efb8432e4e22d